### DISCUSSION TITLE
“can’t find cstdint” error with after upgrade to bindgen `0.66`

### DISCUSSION THREAD

**Discussion by purpleP** [rust-lang/rust-bindgen#2611] [Q&A]
Hi 👋 I have a problem with bindgen `0.66`.

A cross-compilation results in `can’t find cstdint` error. The file exists in sysroot, but clang searches for it in host folders like `/usr/lib`.

```
./audio-codec/audio-codec/include/opencore/amr_const.h:4:10: fatal error: 'cstdint' file not found
```
My environment is debian buster.

`BINDGEN_EXTRA_CLANG_ARGS='--sysroot=/usr/aarch64-linux-gnu -I/usr/include/aarch64-linux-gnu/'`

```
apt install build-essential \
    binutils \
    cmake \
    pkg-config \
    llvm-dev \
    libclang-dev \
    clang \
    curl \
    git \
    git-lfs \
    libssl-dev \
    ca-certificates \
    jq \
    protobuf-compiler
    g++-aarch64-linux-gnu \
    gcc-aarch64-linux-gnu \
    libc6-dev-arm64-cross \
    pkg-config:arm64 \
    libssl-dev:arm64 \
    binutils-aarch64-linux-gnu
```

It was working fine with bindgen `0.54`.
![strace of a build](https://github.com/rust-lang/rust-bindgen/assets/6205841/03d5e53b-d89a-462f-90fb-83ff3cb166d4)

![IMG_2121](https://github.com/rust-lang/rust-bindgen/assets/6205841/986ee39c-a3de-4543-b164-7506205063bb)

**Comment by xulongzhe** [comment]
is there any one can resolve it ?

**Reply by pvdrz** [reply]
I think it is pretty hard for anyone else to track down the cause unless you can provide a minimal example
